### PR TITLE
Add feature flag for disabling encryption in Element Call

### PR DESCRIPTION
**Pull request by toger5**
We want an option to disable the call encryption while it is not supported on all platforms and for specific experiments/demos. This is a very easy way to make this configurable for the ones working on/with it and allows to simplify some experiments.

Should be merged with: https://github.com/vector-im/element-web/pull/26548

## Checklist

-   [x] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
Here's what your changelog entry will look like:

## ✨ Features
 * Add feature flag for disabling encryption in Element Call ([\#11837](https://github.com/matrix-org/matrix-react-sdk/pull/11837)). Contributed by @toger5.

### CODE REVIEW

File: src/i18n/strings/en_EN.json [hunk 1]
```diff
@@ -1398,6 +1398,7 @@
         "element_call_video_rooms": "Element Call video rooms",
         "experimental_description": "Feeling experimental? Try out our latest ideas in development. These features are not finalised; they may be unstable, may change, or may be dropped altogether. <a>Learn more</a>.",
         "experimental_section": "Early previews",
+        "feature_disable_call_per_sender_encryption": "Disable per sender encryption for element call",
```
Comment (richvdh) [inline src/i18n/strings/en_EN.json:1401]:
```suggestion
        "feature_disable_call_per_sender_encryption": "Disable per-sender encryption for Element Call",
```

**Review by richvdh — CHANGES_REQUESTED** [review]
* Documentation at https://github.com/vector-im/element-web/blob/develop/docs/labs.md please!
* Tests please!

This is not an "internal change" and I think it deserves a place in the changelog. I've relabelled it accordingly. Please don't just label everything as "T-Task".

On which note: I've also updated the title of the PR to better reflect what it's doing. Disabling encryption across the whole app would be quite a different thing! In general, please take the time to ensure your PR titles are accurate.

**Review by richvdh — APPROVED** [review]
lgtm otherwise

File: test/models/Call-test.ts [hunk 2]
```diff
@@ -925,6 +925,30 @@ describe("ElementCall", () => {
...
             expect(destroyPersistentWidgetSpy).toHaveBeenCalled();
         });
+
+        it("the perParticipantE2EE url flag is used in encrypted rooms while respecting the feature_disable_call_per_sender_encryption flag", async () => {
+            // We destroy the call created in beforeEach because we test the call creation process.
+            call.destroy();
+            const addWidgetSpy = jest.spyOn(WidgetStore.instance, "addVirtualWidget");
+            // If a room is not encrypted we will never add the perParticipantE2EE flag.
+            client.isRoomEncrypted.mockReturnValue(true);
+            {
```
[the remark this answers is not in the document]
Comment (toger5) [reply test/models/Call-test.ts:935]: I can remove them. (The intention was to visually block out where we have the mocked room encryption)

**Pull request merged**